### PR TITLE
Update TM&B in guides section of docs

### PR DESCRIPTION
**Pull request by krisdale**
- Add disclaimers for Intel trademarks
- Update/correct product names in text per Intel guidance

Signed-off-by: Kristal Dale <kristal.dale@intel.com>

### CODE REVIEW

**Comment by krisdale** [comment]
@mvincerx - @bd-dean needs to confirm the changes in the stacks section as I updated several product names to be inline with Intel trademarks. @bd-dean will confirm updates to stacks pages are correct for the content.

File: source/guides/clear/compatible-kernels.rst [hunk 1]
```diff
@@ -37,7 +37,7 @@ VM only
 Kernel KVM
     The *kernel-kvm* bundle focuses on the Linux
     :abbr:`KVM (Kernel-based Virtual Machine)`. It is optimized for fast
-    booting and performs best on Virtual Machines running on the Intel®
+    booting and performs best on Virtual Machines running on the Intel
```
Comment (mvincerx) [inline source/guides/clear/compatible-kernels.rst:40]: Same as above: "Intel architecture"

File: source/guides/clear/compatible-kernels.rst [hunk 2]
```diff
@@ -49,14 +49,16 @@ Kernel KVM
 Kernel Hyper-V\*
     The *kernel-hyperv* bundle focuses on running Linux on Microsoft\*
     Hyper-V. It is optimized for fast booting and performs best on Virtual
-    Machines running on the Intel® architectures described on the
+    Machines running on the Intel architectures described on the
```
Comment (mvincerx) [inline source/guides/clear/compatible-kernels.rst:52]: Same as above.

File: source/guides/clear/performance.rst [hunk 3]
```diff
@@ -101,7 +101,7 @@ optimize software builds for runtime performance. Some significant flags that
 
 	.. note::
 		|CL| doesn't require Advanced Encryption Standard (AES), so it should
-		run on some Intel CPUs from the first generation of Intel® microarchitecture code name Nehalem (released in 2008). Refer to the
+		run on some Intel CPUs from the first generation of Intel microarchitecture code name Nehalem (released in 2008). Refer to the
```
Comment (mvincerx) [inline source/guides/clear/performance.rst:104]: Revert to original: Intel® microarchitecture code name Nehalem

File: source/guides/maintenance/cpu-performance.rst [hunk 4]
```diff
@@ -248,15 +247,13 @@ The following output means the configuration has already been applied:
...
+For more information, see http://www.intel.com/technology/turboboost*
 
-   Intel® Turbo Boost Technology requires a PC with a processor with Intel
-   Turbo Boost Technology capability. Intel Turbo Boost Technology performance
-   varies depending on hardware, software and overall system configuration.
-   Check with your PC manufacturer on whether your system delivers Intel Turbo
-   Boost Technology. For more information, see http://www.intel.com/technology/turboboost
-
-   Intel SpeedStep is a trademark of Intel Corporation or its subsidiaries.
+*Intel, Intel SpeedStep, and the Intel logo are trademarks of Intel Corporation or its subsidiaries.*
```
Comment (mvincerx) [inline source/guides/maintenance/cpu-performance.rst:256]: SpeedStep can be removed. I don't see it mentioned in the body text.

File: source/guides/stacks/dbrs.rst [hunk 5]
```diff
@@ -14,7 +14,7 @@ Overview
 ********
 
 The Database Reference Stack is integrated, highly-performant, open source,
-and optimized for 2nd generation Intel® Xeon® Scalable Processors and Intel®
+and optimized for 2nd generation Intel® Xeon® Scalable processors and Intel®
```
Comment (mvincerx) [inline source/guides/stacks/dbrs.rst:17]:
```suggestion
and optimized for 2nd generation Intel® Xeon® Scalable processor family and Intel®
```

File: source/guides/stacks/dbrs.rst [hunk 6]
```diff
@@ -52,10 +52,10 @@ The release announcement for each release provides more detail about the stack f
 Hardware Requirements
 *********************
 
-* Intel® Xeon Scalable Platform with Intel® C620 chipset series
-* 2nd Gen Intel® Xeon Scalable processor CPU (Intel® Optane™ PMM-enabled stepping) Provides cache & memory control.  Intel® Optane™  PMem works only on systems powered by 2nd Generation Intel® Xeon® Platinum or Gold processors.
+* Intel Xeon Scalable platform with Intel® C620 series chipset
+* 2nd Gen Intel Xeon Scalable processor CPU (Intel® Optane™ PMem-enabled stepping). Provides cache & memory control.  Intel Optane PMem works only on systems powered by 2nd Generation Intel® Xeon® Platinum or Intel® Xeon® Gold processors.
```
Comment (mvincerx) [inline source/guides/stacks/dbrs.rst:56]:
```suggestion
* 2nd Gen Intel Xeon Scalable processor CPU (Intel® Optane™ PMem-enabled stepping). Provides cache & memory control.  Intel Optane PMem works only on systems powered by 2nd Generation Intel® Xeon® Platinum or the Intel® Xeon® Gold processor.
```

File: source/guides/stacks/dbrs.rst [hunk 7]
```diff
@@ -104,7 +104,7 @@ Firmware configuration
 
    When updating DCPMM Firmware, all DCPMM parts must be in the same mode (you cannot mix 1LM and 2LM parts).
 
-The latest firmware download for the Intel® Server System S2600WF Family is available at the `Intel Download Center`_
+The latest firmware download for the Intel Server System S2600WF is available at the `Intel Download Center`_
```
Comment (mvincerx) [inline source/guides/stacks/dbrs.rst:107]: Revert to original: Intel® Server System, per DB.
Yes on removing "Family"

File: source/guides/stacks/dlrs-inference.rst [hunk 8]
```diff
@@ -19,7 +19,7 @@ Overview
 
 The solution covered here requires the following software components:
 
-* `Deep Learning Reference Stack`_ which is a |CL-ATTR| based Docker\* container providing deep learning frameworks and is optimized for Intel Xeon Scalable platforms.
+* `Deep Learning Reference Stack`_ which is a |CL-ATTR| based Docker\* container providing deep learning frameworks and is optimized for Intel® Xeon® platforms.
```
Comment (mvincerx) [inline source/guides/stacks/dlrs-inference.rst:22]:
```suggestion
* `Deep Learning Reference Stack`_ which is a |CL-ATTR| based Docker\* container providing deep learning frameworks and is optimized for the Intel® Xeon® platform.
```

File: source/guides/stacks/dlrs-inference.rst [hunk 9]
```diff
@@ -85,7 +85,7 @@ Although this guide assumes a |CL| host system, it has also been validated with
 Recommended Hardware
 ====================
 
-We validated this guide on an `Intel Cascade Lake`_ server and this is recommended to get optimal performance and take advantage of the built in  Intel® Deep Learning Boost functionality.
+We validated this guide on a server with a 2nd Generation Intel Xeon Scalable processor, formerly Cascade Lake, and this is recommended to get optimal performance and take advantage of the built in Intel® Deep Learning Boost (Intel® DL Boost) functionality.
```
Comment (mvincerx) [inline source/guides/stacks/dlrs-inference.rst:88]:
```suggestion
We validated this guide on a server with a 2nd Generation  Intel® Xeon® Scalable processor, formerly Cascade Lake, and this is recommended to get optimal performance and take advantage of the built in Intel® Deep Learning Boost (Intel® DL Boost) functionality.
```

Despite the fact that this doc references  "Intel® Xeon® platform", above, the "Intel® Xeon® Scalable processor" is a unique phrase whose meaning is slightly different. One is general; the other is more specific.

File: source/guides/clear/compatible-kernels.rst [hunk 10]
```diff
@@ -24,7 +24,7 @@ Also compatible with VMs
 Kernel LTS
     The *kernel-lts* bundle focuses on the bare metal platforms but uses the
     latest :abbr:`LTS (Long Term Support)` Linux kernel. It is optimized for
-    fast booting and performs best on the Intel® architectures described on the
+    fast booting and performs best on the Intel architectures described on the
```
[the remark this answers is not in the document]
Comment (krisdale) [reply source/guides/clear/compatible-kernels.rst:27]: Will update this to be 'Intel processors' - which is both more consistent with language used on the referenced link and addresses confusing use of 'Intel architecture' (intentionally w/out the (r) b/c its not referencing "Intel(r) architecture", but rather a more general 'this set of Intel architectures..')

File: source/guides/maintenance/cpu-performance.rst (hunk 4 above)
Comment (krisdale) [reply source/guides/maintenance/cpu-performance.rst:256]: Mentioned in section 'P-states (performance states)'

File: source/guides/stacks/dbrs.rst (hunk 5 above)
Comment (krisdale) [reply source/guides/stacks/dbrs.rst:17]: I believe we are good to use 'processors' per the sample usage chart provided in the names DB (which shows an example of 'processor' pluralized).

File: source/guides/stacks/dbrs.rst (hunk 6 above)
Comment (krisdale) [reply source/guides/stacks/dbrs.rst:56]: I believe we are good to use 'processors' per the sample usage chart provided in the names DB (which shows an example of 'processor' pluralized).

File: source/guides/stacks/dbrs.rst (hunk 7 above)
Comment (krisdale) [reply source/guides/stacks/dbrs.rst:107]: I think we can leave the (r) off as this is second use of name on the page. The specific system (s2600wf) isn't part of the name.

File: source/guides/clear/performance.rst (hunk 3 above)
Comment (krisdale) [reply source/guides/clear/performance.rst:104]: Done

File: source/guides/stacks/dlrs-inference.rst (hunk 9 above)
Comment (krisdale) [reply source/guides/stacks/dlrs-inference.rst:88]: I didn't add in the (r)s b/c this is second use of the trademark on the page (The trademark part is 'Intel® Xeon®' which is used as an adjective with one of the approved nouns - as I read this description from the DB, this is the second use of the trademark on the page... glad to discuss if needed :) ). 

I see I did accidentally remove the original link tho! Will add back in.

File: source/guides/stacks/dlrs-inference.rst (hunk 8 above)
Comment (krisdale) [reply source/guides/stacks/dlrs-inference.rst:22]: I believe we are good to use 'processors' per the sample usage chart provided in the names DB (which shows an example of 'processor' pluralized).

File: source/guides/stacks/dlrs-inference.rst (hunk 9 above)
Comment (krisdale) [reply source/guides/stacks/dlrs-inference.rst]: @mvincerx Thanks for the detailed review! Applied updates - I didn't make all changes, based on current working knowledge of TM&B rules (from legal + DB). I explained my reasoning inline w/ your original feedback.

File: source/guides/clear/compatible-kernels.rst [hunk 11]
```diff
@@ -12,7 +12,7 @@ Bare metal only
 
 Kernel native
     The *kernel-native* bundle focuses on the bare metal platforms. It is
-    optimized for fast booting and performs best on the Intel® architectures
+    optimized for fast booting and performs best on the Intel processors
```
Comment (mvincerx) [inline source/guides/clear/compatible-kernels.rst:15]: Correct this per the DB.  If you disagree, show me the example where the phrase "Intel processor" originates.  
Intel® Architecture Processors

File: source/guides/clear/compatible-kernels.rst [hunk 12]
```diff
@@ -24,7 +24,7 @@ Also compatible with VMs
 Kernel LTS
     The *kernel-lts* bundle focuses on the bare metal platforms but uses the
     latest :abbr:`LTS (Long Term Support)` Linux kernel. It is optimized for
-    fast booting and performs best on the Intel® architectures described on the
+    fast booting and performs best on the Intel processors described on the
```
Comment (mvincerx) [inline source/guides/clear/compatible-kernels.rst:27]: Intel Architecture Processors

**Pull request merged**